docs/user_guide/edge_development/index.rst

```diff
@@ -117,6 +117,8 @@ These steps can be best illustrated as a finite state machine as follows:
 .. image:: ../../resources/edge_device_finite_state_machine.png
     :height: 500px
 
+.. _device_simulation:
```

Comment:
I thought you are going to include the following link ?  

https://nvflare.readthedocs.io/en/main/user_guide/edge_development/index.html#simulator-configuration

Reply:
yes, it will jump, but the device simulation place does not include above link. Can you double check